src/Workspaces/Core/Portable/FindSymbols/Declarations/DeclarationFinder.cs

```diff
@@ -58,7 +58,7 @@ internal static partial class DeclarationFinder
                 // use the DeclarationOnlyCompilation if we have one, avoiding needing to build the
                 // full compilation if we don't have that.
                 var containsSymbol = isExactNameSearch
-                    ? await project.ContainsSymbolsWithNameAsync(query.Name, filter, cancellationToken).ConfigureAwait(false)
```

Comment:
Comment why you're not using other overload that takes predicate + SymbolFilter? I assume that's because the "probably contains name" is a cheaper check and not likely to hit many false positives, but it'd be good to write a comment here.

Reply:
will do